### PR TITLE
README Rework

### PR DESCRIPTION
**Pull request by ElectronicsArchiver**
- Formatting / Refactoring
- Separation Of Content
- Fixes, Links, Badges

### CODE REVIEW

File: docs/Limitations.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+It is possible to emulate an index controller while <br>
+providing your own input profiles and bindings, <br>
+but we have chosen not to include that by default <br>
+in the driver, as to preserve compatibility with <br>
+default index controller bindings.
+
+**➔**  *Implementing a custom controller can be achieved by* <br>
+     *creating a derivative class of [`DeviceDriver`] in which* <br>
+     *you specify the properties & settings of your controller.*
```
Comment (danwillm) [inline docs/Limitations.md:24]:
```suggestion
If you want to implement your own device and use the utilities that OpenGloves provides (such as bone calculations and communication), create your own class that is derived from [`DeviceDriver`], and implement `StartingDevice`, `SetupProps`, `HandleInput` and `StoppingDevice`.
```

File: docs/Limitations.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+
+**➔**  An example of a fully custom controller is **[LucidGloveDriver]** <br>
+     *which you are free to adapt to your needs.*
+
+<br>
+
+#### Dynamic Inputs
+
+Due to how **OpenVR** wworks, inputs cannot be set ***dynamically***.
```
Comment (danwillm) [inline docs/Limitations.md:33]: `wworks`.

File: docs/Limitations.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+Our inputs for `Index Controller Emulated Devices` are ***fixed***.
+
+**➔**  *However, you can define your own inputs in a* <br>
+     *custom device with a different input profile.*
```
Comment (danwillm) [inline docs/Limitations.md:38]: can we reorder this, just so it makes a little more sense?
```suggestion
Our inputs for `Index Controller Emulated Devices` are ***fixed*** to that of the index controller, and cannot have custom inputs.

**➔**  *However, you can define your own inputs in a* <br>
     ***custom device** with a different input profile.*
```

File: README.md [hunk 4]
```diff
@@ -1,80 +1,83 @@
-# OpenGloves Driver
+# OpenGloves Driver [![Steam Badge]][Steam] [![Discord Badge]][Discord]
 
-OpenGloves is an OpenVR Driver for Virtual Reality (DIY) hardware, specifically designed to be used with VR Gloves.
+A general purpose **SteamVR** driver for **DIY** VR <br>
+Glove hardware providing an **[Open Interface]**.
```
Comment (danwillm) [inline README.md:4]:
```suggestion
A general purpose **SteamVR** driver for **DIY** VR <br>
(Glove) hardware, which provides an **[Open Interface]** for anyone wanting to connect their own hardware to SteamVR,
```

File: README.md [hunk 5]
```diff
@@ -1,80 +1,83 @@
...
 
-## Installation and Usage
+- **Tracker / Controller Positioning** <br>
+    *Automatic relative calibration* <br>
+    *Controller ⟷ Glove*
 
-We strongly recommend downloading the driver from Steam, to recieve automatic updates and UI settings.  
+- **Button / Joystick Inputs** <br>
+    *➜ Trigger | A | B* <br>
+    *➜ Joystick | X | Y*
```
Comment (danwillm) [inline README.md:26]: joysticks also have a click
```suggestion
    *➜ Joystick | X | Y | Click*
```

File: README.md [hunk 6]
```diff
@@ -1,80 +1,83 @@
...
+<br>
 
-### Building
-If you plan on making changes to the driver, use the instructions in [Building.md](https://github.com/LucidVR/opengloves-driver/blob/develop/BUILDING.md) to build and install the Debug version of the driver to SteamVR.
+### Steam UI
 
-## Compatibility
-### Officially Compatible Hardware
-While it is possible for any hardware to be compatible with OpenGloves, the driver is known to be compatible with these projects:
+The **[Steam]** version of **OpenGloves** also provides you a **[UI]**.
```
Comment (danwillm) [inline README.md:37]: Perhaps?
```suggestion
When **OpenGloves** is downloaded on **[Steam]**, it comes with a **[UI]** for configuring driver related features.
```

File: docs/Compatibility.md [hunk 7]
```diff
@@ -0,0 +1,75 @@
...
+
+- **[LucidGloves]** <br>
+    *by [Lucas VRTech]*
+
+- **[Fngrs]** <br>
+    *by [danwillm]*
+
+<br>
+
+**You made your own hardware, let us know!**
```
Comment (danwillm) [inline docs/Compatibility.md:18]:
```suggestion
 **Made your own hardware? Let us know!**
```

File: docs/Compatibility.md [hunk 8]
```diff
@@ -0,0 +1,75 @@
...
+inputs from a variety of VR controllers, <br>
+it is primarily designed for VR Gloves.
+
+<br>
+
+#### Custom Hardware
+
+For making your own hardware compatible, <br>
+please refer to the **[Driver Input]** page, listing <br>
+`Encoding Schemes` & `Communication Methods`.
```
Comment (danwillm) [inline docs/Compatibility.md:34]:
```suggestion
To make your own hardware compatible with **OpenGloves**, <br>
refer to the **[Driver Input]** page, which <br>
provides information on making your hardware compatible.

File: docs/Compatibility.md [hunk 9]
```diff
@@ -0,0 +1,75 @@
...
+
+<br>
+
+#### LucidVR
+
+Our own **[Firmware]** for the **OpenGloves** is <br>
```
Comment (danwillm) [inline docs/Compatibility.md:40]:
```suggestion
Our own **[Firmware]** that is comatible with **OpenGloves** is <br>

File: docs/Compatibility.md [hunk 10]
```diff
@@ -0,0 +1,75 @@
...
+
+##### Finger Curling
+
+Is supported in games that work with the **Index** controllers.
+
+<br>
+
+##### Force Feedback
+
+Unfortunately this selection is **[More Limited]**.
```
Comment (danwillm) [inline docs/Compatibility.md:59]:
```suggestion
Unfortunately, game compatibility with Force Feedback is **[More Limited]**.<br>
If you'd like to make your game/mod compatible with Force Feedback, refer to this wiki page
```
(+ link to https://github.com/LucidVR/opengloves-driver/wiki/Integrating-Force-Feedback)

**Review by danwillm — CHANGES_REQUESTED** [review]
Thanks for this, just a few comments 👍

**Review by danwillm — APPROVED** [review]

**Review by lucas-vrtech — APPROVED** [review]
Looks great :)

**Pull request merged**